cmd/environment_variables.go

```diff
@@ -18,6 +19,23 @@ func SetFlagsFromEnvVariables(fs *pflag.FlagSet) {
 		envVar := flagToEnvVarName(f)
 		if val, present := os.LookupEnv(envVar); present {
 			fs.Set(f.Name, val)
+			return
+		}
+
+		// Do not look for _FILE if the application is already expecting a file.
+		if strings.HasSuffix(envVar, "_FILE") {
+			return
+		}
+
+		val, present := os.LookupEnv(envVar + "_FILE")
+		if present {
```

Comment:
Could you inline the assignment within the if condition? (see line 20)

Reply:
Heh. I had it like that, but was messing around with things, and must have forgotten to move it back! Will do!